docs/how-to/pacemaker.md

```diff
@@ -0,0 +1,390 @@
+```sh
+[root@pmm1 ~]# hostnamectl hostname pmm1
+
+[root@pmm1 ~]# cat /etc/hosts
+127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
+::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+192.168.122.151 pmm1 
+192.168.122.126 pmm2
+
+[root@pmm2 ~]# hostnamectl hostname pmm2
+
+[root@pmm2 ~]# cat /etc/hosts
+127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
+::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+192.168.122.126 pmm2
+192.168.122.151 pmm1
+```
+
+## Configure an LVM volume with an ext4 file system
+
+This procedure is carried out on one node that we would initially assign as Active. In this example, it is  `pmm1` node.
+
+```sh
+pvcreate /dev/sda1
+  Physical volume "/dev/sda1" successfully created.
+  Creating devices file /etc/lvm/devices/system.devices
+
+vgcreate --setautoactivation n pmm-server /dev/sda1
+  Volume group "pmm-server" successfully created with system ID pmm1
+
+vgs -o+systemid
+  VG         #PV #LV #SN Attr   VSize   VFree   System ID
+  pmm-server   1   0   0 wz--n- <20.00g <20.00g pmm1     
+
+lvcreate -l100%VG -n srv pmm-server
+  Logical volume "srv" created.
+
+lvs
+  LV   VG         Attr       LSize   Pool Origin Data%  Meta%  Move Log Cpy%Sync Convert
+  srv  pmm-server -wi-a----- <20.00g                                                    
+
+mkfs.ext4 /dev/pmm-server/srv
+mke2fs 1.46.5 (30-Dec-2021)
+Discarding device blocks: done                            
+Creating filesystem with 5241856 4k blocks and 1310720 inodes
+Filesystem UUID: 2db9af47-f0b8-4724-980b-52b478750d76
+Superblock backups stored on blocks: 
+	32768, 98304, 163840, 229376, 294912, 819200, 884736, 1605632, 2654208, 
+	4096000
+
+Allocating group tables: done                            
+Writing inode tables: done                            
+Creating journal (32768 blocks): done
+Writing superblocks and filesystem accounting information: done   
+
+mkdir -p /mnt/pmm-server/srv
+mount /dev/pmm-server/srv /mnt/pmm-server/srv/
+restorecon -R /mnt/pmm-server/srv/
+
+```
+
+## Create HA cluster
+
+Authorize the nodes and create a cluster:
+```
+pcs host auth pmm1 pmm2
+
+pcs cluster setup pmm_ha_cluster --start pmm1 pmm2
+
+pcs cluster status
+```
+
+Disabling fencing only for demo purposes:
+```
+pcs property set stonith-enabled=false
+```
+
+!!! caution alert alert-warning "Important"
+    Ensure that  you configure proper fencing on a production cluster.
+
+## Add resources to the cluster
+
+With the following commands, we will add:
+
+* LVM HA that would be properly migrated and mounted by Pacemaker
+* VirtualIP that also would be migrated and added by the Pacemaker to the Active node
+* PMM Server systemd service that would start PMM Server with correct resources in place
+
+```sh
+pcs resource create pmm_lvm ocf:heartbeat:LVM-activate vgname=pmm_server vg_access_mode=system_id --group pmm
```

Comment:
May be typo here 'pmm_server'? Because in 'vgcreate --setautoactivation n pmm-server /dev/sda1' is 'pmm-server'.

Reply:
fixed! tnx!